style.css

```diff
@@ -454,3 +535,135 @@ textarea {
 .btn3:hover {
   background-color: #fff;
 }
+
+@media screen and (min-width: 768px) {
+  .hamburger {
+    display: none;
+  }
+
+  .container {
+    flex: 1;
+  }
+
+  .container .head {
+    width: 720px;
+    padding: 0;
+    padding-left: 300px;
+    padding-top: 100px;
+    margin: 0;
+  }
+
+  .container .pr {
+    width: 650px;
+    height: 90px;
+    padding: 0;
+    padding-left: 300px;
+    margin: 0;
+  }
+
+  .container .a {
+    padding: 0;
+    margin: 0;
+    padding-left: 300px;
+    padding-bottom: 10px;
+  }
+
+  .container .social-media-container {
+    padding-left: 300px;
+    margin: 0;
+    padding-bottom: 200px;
+  }
+
+  .container_last .social-media-container {
+    margin: 0;
+  }
+
+  .Job-container {
+    width: 1000px;
+    margin-left: 100px;
+    grid-template-columns: auto auto;
+    gap: 10px;
+    justify-content: center;
+  }
```

Comment:
- [ ] Your job-container fixed width is causing the card to be cut out of the screen in some screen sizes. Kindly fix that.
![image](https://user-images.githubusercontent.com/61919617/172725382-ce275bc5-ac01-46a5-804e-be30637af78c.png)